client/app/pages/query-snippets/QuerySnippetsList.less

```diff
@@ -8,3 +8,16 @@
   -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
 }
+
+.ant-table-row {
+  height: 1px;
```

Comment:
That's a surprising one. Why is that? Also, this looks like an unqualified selector -- will these three classnames impact all `ant-table-cell`s, `ant-table-row`s, etc.?